public/locales/en.json

```diff
@@ -109,6 +109,7 @@
     "membershipRequests": "Membership Requests",
     "deleteOrganization": "Delete Organization",
     "deleteMsg": "Do you want to delete this organization?",
+    "confirmDeleteText": "Please type <bold>{{orgName}}</bold> to confirm",
```

Comment:
You need to make this work in all languages. Use Google Translate for the others.

Reply:
> Regarding your second question, the YES button remains disabled as long as the user types a wrong organization name in the input field or leaves the input field empty, the YES button is only enabled when the user get the organization name correctly and its also case sensitive.

I mean style the button and make the color faint on disabled mode. Maybe make it less green.

Reply:
@palisadoes i have made the required changes, added language support for the languages and also made use of debounce input.